source/extensions/filters/http/jwt_authn/filter.cc

```diff
@@ -38,13 +40,13 @@ Http::FilterHeadersStatus Filter::decodeHeaders(Http::HeaderMap& headers, bool)
   return Http::FilterHeadersStatus::StopIteration;
 }
 
-void Filter::onComplete(const Status& status) {
+void Filter::onComplete(const Status& status, VerifyContext&) {
```

Comment:
Can we NOT pass in VerifyContext& in the onComplete() callback.   There is only one VerifyContext per request.

Reply:
removed from VerifierCallback onComplete.